### PR TITLE
fix(icon, menuBar, slider, tabs, util): use CSP compliant element.style…

### LINKED ISSUE #6602: Inline <style> elements violates style-src Content Security Policy
See https://developer.mozilla.org/en-US/docs/Web/Security/CSP/CSP_policy_directives#style-src

In order to address this, there would need to be build-time tooling to let people generate their own theme css outputs.

### PR DESCRIPTION
**Pull request by Emeegeemee**
…instead of style attribute

Fixes #6602

### CODE REVIEW

File: src/components/tabs/js/tabsController.js [hunk 1]
```diff
@@ -163,6 +163,9 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
 
   function handleMaxTabWidth (newWidth, oldWidth) {
     if (newWidth !== oldWidth) {
+      angular.forEach(elements.tabs, function(tab) {
+        tab.style.maxWidth = newWidth + 'px';
```
Comment (jelbourn) [inline src/components/tabs/js/tabsController.js]: Does this update if the max-width changes? 

(I'm not actually sure if it needs to. cc @robertmesserle)

File: src/core/util/util.js [hunk 2]
```diff
@@ -321,7 +322,14 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
     },
     floatingScrollbars: function() {
       if (this.floatingScrollbars.cached === undefined) {
-        var tempNode = angular.element('<div style="width: 100%; z-index: -1; position: absolute; height: 35px; overflow-y: scroll"><div style="height: 60px;"></div></div>');
+        var tempNode = angular.element('<div><div></div></div>');
```
Comment (jelbourn) [inline src/core/util/util.js]: I believe you could write this as:
```js
var tempNode = angular.element('<div><div></div></div>').css({
  width: '100%',
  'z-index': -1,
  ...
});
```

File: src/components/tabs/js/tabsController.js (hunk 1 above)
Comment (Emeegeemee) [inline src/components/tabs/js/tabsController.js]: It does, handleMaxTabWidth is the setter function for the maxTabWidth property on the [ctrl](https://github.com/Emeegeemee/material/blob/619a7a3adffa05e1340157792ee0064cf1b74ac7/src/components/tabs/js/tabsController.js#L25). It does need to handle the max width changing or else the tabs content get cut off, from my manual testing.

File: src/core/util/util.js (hunk 2 above)
Comment (Emeegeemee) [inline src/core/util/util.js]: Done and updated all css() calls.

**Comment by jelbourn** [comment]
LGTM

**Comment by jelbourn** [comment]
Thanks for the contribution! It should be part of the 1.0.3 release later this week.

**Pull request closed without merging**